MAINTAINERS.md

```diff
@@ -222,104 +180,15 @@ we follow [Keep a CHANGELOG](http://keepachangelog.com/). The format is simple:
-
-> #### Configure your checkout
->
-> Another way you can simplify your job is to configure your checkout. You can omit the need to type
-> the `--no-ff` flag on each merge by adding the following configuration line to the branch
-> configuration in the checkout's `.git/config` file:
->
-> ```dosini
-> [branch "master"]
->     ...
->     mergeoptions = --no-ff
-> ```
-
-As an example of merging a hotfix:
+noted; things like bugfixes, coding standards fixes, continuous integration changes, or typo fixes do not need
+to be communicated. However, anything that falls under an addition, deprecation, removal
+MUST be noted. Please provide a succinct but *narrative* description for the change you merge.
 
-```console
-$ git checkout master
-$ git merge --edit --no-ff hotfix/2854
-$ git checkout develop
-$ git merge --edit --no-ff hotfix/2854
-```
-
-Note that, because this is a hotfix, it was merged to both `master` *and* `develop`!
-
-As an example of merging only a feature:
-
-```
-$ git checkout develop
-$ git merge --edit --no-ff feature/2719
-```
-
-Since this is a feature, it was merged to `develop` *only*.
-
-The commit message for the target branch should include the merge statement, as well as a line like
-the following:
-
-```
-Close #2854
-```
-
-If merging to both `master` and `develop`, the `develop` branch should indicate that it's *forward
-porting* a fix:
-
-```
-Forward port #2854
-```
+You may also ask the contributor to add these entries to get a feeling for what they're trying to accomplish.
 
-If the fix will close other PRs or issues, also close those in the target branch using the
-`Fix #...` or `Close #...` notation.
+## Performing a Release
 
-> #### Make macros for commit messages
->
-> Automate, automate, automate. Create a macro in the editor you have configured git to invoke for
-> each of these commit messages; that way you can hit a key combination to fill in the messages for
-> you.
-
-### Push the changes upstream
-
-At this point, you can push your changes. If you committed a bugfix, push both branches:
-
-```console
-$ git push origin master:master && git push origin develop:develop
-```
-
-If committing a feature, push only the `develop` branch:
-
-```console
-$ git push origin develop:develop
-```
-
-Once pushed, you can delete the merge branch you created.
-
-> ### Conflicts
->
-> Occasionally, more than one maintainer will be merging to the same repository. As such, do a quick
-> `git fetch origin` prior to pushing to check for changes. If you see any, you'll need to reset
-> your branches, and re-merge:
->
-> ```console
-> $ git checkout develop && git reset --hard origin/develop
-> $ git checkout master && git reset --hard origin/master
-> $ git merge --no-ff hotfix/2719
-> $ git checkout develop
-> $ git merge --no-ff hotfix/2719
-> ```
->
-> If you notice merge conflicts, delete your merge branch, and start again from the top.
+Releases are automatically performed by closing a milestone.
```

Comment:
```suggestion
Releases are automatically performed by closing a milestone. All issues and pull
requests in the milestone MUST be closed before closing the milestone; otherwise,
the release will not be issued.
```